docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,332 @@
+    * Check the version of the different Substra components:
+
+      * if you are using a release you can use :ref:`the compatibility table <additional/release:Compatibility table>`.
+      * if you are using the latest commit from the ``main`` git branch, check that you are up-to-date and see if there were any open issue in the repositories or any bugfixes in the latest commits.
+
+   You can also go through :doc:`the instructions one more time </operations/getting-started>`, maybe they changed since you last saw them.
+
+Troubleshooting prerequisites
+-----------------------------
+
+This section summarize errors happening when you are not meeting the hardware requirements. Please check if `you match these <#hardware>`__ first.
+
+.. note::
+   The instructions are targeted to some specific platforms (Docker for Windows in certain cases and Docker for Mac), where you can set the resources allowed to Docker in the configuration panel (information available `here for Mac <https://docs.docker.com/desktop/settings/mac/>`__ and `here for Windows <https://docs.docker.com/desktop/settings/windows/>`__).
+
+
+The following list describes errors that have already occurred, and their resolutions.
+
+* .. code-block:: pycon
+
+     <ERROR:substra.sdk.backends.remote.rest_client:Requests error status 502: <html>
+     <head><title>502 Bad Gateway</title></head>
+     <body>
+     <center><h1>502 Bad Gateway</h1></center>
+     <hr><center>nginx</center>
+     </body>
+     </html>
+
+     WARNING:root:Function _request failed: retrying in 1s>
+
+   You may have to increase the number of CPU available in the settings panel.
+
+* .. code-block:: go
+
+     Unable to connect to the server: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
+
+  .. code-block:: go
+
+     Unable to connect to the server: net/http: TLS handshake timeout
+
+  You may have to increase the RAM available in the settings panel.
+
+* If you've got a task with ``FAILED`` status and the logs in the worker are of this form:
+
+  .. code-block:: py3
+
+     substrapp.exceptions.PodReadinessTimeoutError: Pod substra.ai/pod-name=substra-***-compute-*** failed to reach the \"Running\" phase after 300 seconds."
+
+  Your Docker disk image might be full, increase it or clean it with ``docker system prune -a``
+
+Troubleshooting deployment
+--------------------------
+
+Skaffold version 1.31.0
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Status check is broken in version 1.31.0 and kubectl secret manifests are not applied until helm deploy is done, but helm deploy depends on kubectl secret manifests.
+It has been fixed in `Skaffold 1.32.0 (PR #6574) <https://github.com/GoogleContainerTools/skaffold/releases/tag/v1.32.0>`__.
+
+The solution for the version 1.31.0 is to add ``--status-check=false`` when running Skaffold:
+
+.. code-block:: bash
+
+   skaffold dev/run/deploy --status-check=false
```

Comment:
@AlexandrePicosson Based on your feedback on PR https://github.com/Substra/substra-documentation/pull/193 , it looks like to me I should remove this subsection. Do you agree?